tools/test-exists.js

```diff
@@ -2,9 +2,11 @@
  * Sanity check that all repos in test.json actually exist on GitHub
  */
 
-const get = require('simple-get')
-const testPkgs = require('../test/test.json')
-const series = require('run-series')
+import get from 'simple-get'
+import series from 'run-series'
+import { readFileSync } from 'fs'
+
+const testPkgs = JSON.parse(readFileSync('../test/test.json', 'utf8'))
```

Comment:
The previous `require` call was relative to this file, but this is relative to the CWD (I think?)

Also, it seems like the file is in a folder called `external`? 🤔 

```suggestion
const testPkgs = JSON.parse(readFileSync(new URL('../test/external/test.json', import.meta.url), 'utf8'))
```

Tested this:

```sh
> new URL('../test/external/test.json', 'file:///home/user/standard/tools/test-exists.js')
URL {
  href: 'file:///home/user/standard/test/external/test.json',
  ...
}
```